python/chemiscope/structures/_stk.py

```diff
@@ -13,22 +13,24 @@ class Molecule:
     HAVE_STK = False
 
 
-def _stk_valid_structures(
-    frames: Union[Molecule, List[Molecule]],
-) -> Tuple[List[Molecule], bool]:
-    if HAVE_STK and isinstance(frames, Molecule):
-        # deal with the user passing a single frame
-        return [frames], True
-    elif HAVE_STK and isinstance(frames, list) and isinstance(frames[0], Molecule):
-        for frame in frames:
-            assert isinstance(frame, Molecule)
-        return frames, True
+def _stk_valid_structures(structures: Union[Molecule, List[Molecule]]):
```

Comment:
Why not having return type here?

Reply:
No reason, although we don't have it elsewhere in the code =) I can add it back!